src/components/document/student-group-view.tsx

```diff
@@ -1,9 +1,11 @@
 import React, { useState } from "react";
 import { GroupUserModelType } from "src/models/stores/groups";
```

Comment:
I know this wasn't modified in this PR, but this should be a relative path.

Reply:
Changed.